discord/webhook/sync.py

```diff
@@ -45,8 +45,6 @@
     Literal,
     Optional,
     Tuple,
-    Type,
-    TypeVar,
```

Comment:
You should make changes that are related to the main topic of this PR
This is not related to the commands system at all

Reply:
Moved all the extra import stuff to #1608